lib/rules/redux-actions-return-objects.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Prohibits returning anything but objects from Redux actions',
+    },
+    fixable: false,
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          whiteList: {
+            default: [],
+            type: 'array',
```

Comment:
Could it be a `Set`? I don't actually know if `eslint` supports it, but based on the code below, a `Set` would be ideal for doing `O(1)` inclusion checks.

The array is probably small enough that it doesn't matter much, though.

Reply:
I was a little worried about it being slow, but even after adding the 100+ file whitelist (which should never grow in a size), it didn't show up in the top 10 slowest rules from running `TIMING=1 yarn lint`, and the 10th on that list is less than a second for the whole project.

That said, using a set is interesting so I'll see if it's possible. I'd guess it's not since the config for these rules needs to be describable in JSON.

Reply:
`eslint` is limited enough that I suspect it is not possible as well; although, there's always the possibility of using a hash to emulate a set:

```js
whitelist: {
  'filename.js': true,
  'another_filename.js': true,
  ...
}
```

However, if it's not a problem when you profile it, it's not worth doing something like that.